give-a-talk.md

```diff
@@ -1,40 +1,30 @@
 ---
 layout: page
-title: Give a Lightning Talk
+title: We want you to give a talk!
 ---
 {% include JB/setup %}
 
-### We want you to give a talk!
-
-Have you been learning JavaScript and want to spend a couple minutes sharing something you learned?
-
-Did you find a library or discover a JS feature you'd like to share?
-
-Have an interesting method for learning or teaching JavaScript?
+If you have something JavaScript-related to share with your peers that you could say/show in less than 5 minutes, **we want to hear you give a talk**!
 
-Want to share something you made?
+Have you been learning JavaScript and want to spend a couple minutes sharing something you learned? Did you find a library or discover a JS feature you'd like to share? Have an interesting method for learning or teaching JavaScript? Want to share something you made? Want to show off a niche feature of one of your favorite libraries?
 
-Want to show off a niche feature of one of your favorite libraries?
+We also have a wiki page with [talk ideas][1] that you can use for inspiration.
 
-If you have something JavaScript-related to share with your peers that you could say/show in less than 5 minutes, **we want to hear you give a talk**!
 
-### How to give a talk
+#### Submitting your talk
 
-Are you thinking of giving a short lightning talk?
+Once you are ready to submit a talk please email your talk subject to the [SDJS organizers](mailto:sandiegojs-organizers@googlegroups.com).
 
-Comment on an event page or, better yet, contact
-[the organizers](mailto:sandiegojs-organizers@googlegroups.com) directly.
+#### Preparing for Your Talk
 
-Don't know what you want to talk about yet?
+You will be presenting on a `600px` width projector so make sure to tailor your slides to that dimension. Using a large font with white text on a black background will make your content more legible.
 
-We have a wiki page with [talk ideas][] that you can use for inspiration.
+Want to chat about your talk while you prepare it?  Join the [chat room][2]!
 
-Want to chat about your talk while you prepare it?  Join [the chat room](/chat-room.html)!
 
```

Comment:
Maybe we should have a do's and don'ts section.

- Do use black on white slides
- Don't use more than a small number of slides (maybe 6 preferably?)
- Don't try to do a live demo or write code or do anything but slides for your first talk... it's really hard for a lightning talk

We could also link to lightning talks we think are good examples.